mix.exs

```diff
@@ -77,6 +77,9 @@ defmodule Explorer.MixProject do
           Explorer.Backend.LazyFrame,
           Explorer.Backend.LazySeries,
           Explorer.PolarsBackend
+        ],
+        Internal: [
+          Explorer.Duration
```

Comment:
I'm not sure about this division, but I wanted to keep this separated. I chose to document the module because we refer to it in many places - mostly typespecs.

Reply:
I like this approach.

Reply:
Especially because if a user does `to_list` to a duration series, they'll see these structs.

Reply:
@billylanchantin thanks! And sorry for merging it too fast :grimacing: 
If you find anything here, please let me know so I can fix before the release.

Reply:
No no it looks great!